sdk/formrecognizer/Azure.AI.FormRecognizer/src/CustomFormModel.cs

```diff
@@ -112,29 +123,16 @@ private static IReadOnlyList<CustomFormSubmodel> ConvertFromLabeled(Model model)
         private static IReadOnlyList<TrainingDocumentInfo> ConvertToTrainingDocuments(TrainResult trainResult)
         {
             var trainingDocs = new List<TrainingDocumentInfo>();
-            if (trainResult?.TrainingDocuments != null)
+            foreach (var docs in trainResult.TrainingDocuments)
             {
-                foreach (var docs in trainResult?.TrainingDocuments)
-                {
-                    trainingDocs.Add(
-                        new TrainingDocumentInfo(
-                            docs.DocumentName,
-                            docs.PageCount,
-                            docs.Errors ?? new List<FormRecognizerError>(),
-                            docs.Status));
-                }
+                trainingDocs.Add(
+                    new TrainingDocumentInfo(
+                        docs.DocumentName,
+                        docs.PageCount,
+                        docs.Errors ?? new List<FormRecognizerError>(),
+                        docs.Status));
             }
             return trainingDocs;
         }
-
-        private static IReadOnlyList<FormRecognizerError> ConvertToFormRecognizerError(TrainResult trainResult)
-        {
-            var errors = new List<FormRecognizerError>();
-            foreach (var error in trainResult?.Errors)
-            {
-                errors.Add(new FormRecognizerError(error.ErrorCode, error.Message));
-            }
-            return errors;
-        }
```

Comment:
Removing this method since we're literally just copying things from one list to another. We can simply do:
```cs
Errors = model.TrainResult.Errors;
```
in the constructor.

Reply:
Added some null checks as well.